docs/my-website/docs/providers/huggingface.md

```diff
@@ -4,42 +4,42 @@ import TabItem from '@theme/TabItem';
 
 # Huggingface
 
-LiteLLM supports the following types of Huggingface models:
-* Text-generation-interface: [Here's all the models that use this format](https://huggingface.co/models?other=text-generation-inference).
```

Comment:
It's okay to call out the dedicated endpoints, but it's more useful for me to know what the api formats supported are. 

TGI is an API format, so it's a useful callout.

Reply:
Ok I updated, fyi all Text-generation models via HF Inference now use TGI. And conversational-task is depreciated.